kinto/core/initialization.py

```diff
@@ -101,6 +101,15 @@ def setup_authentication(config):
     from configuration.
     """
     config.include('pyramid_multiauth')
+    settings = config.get_settings()
+
+    policies = aslist(settings['multiauth.policies'])
+    if 'basicauth' in policies:
```

Comment:
That's a really interesting question, this mean adding it in kinto.plugins as well which doesn't feel right to me, I can see multiple way of activating the `include()` while disabling it in the `multiauth.policies` which is why I didn't took that path.